### PR TITLE
ISSUE: #168

### PR DESCRIPTION
**Pull request by xam8re**

### CODE REVIEW

**Comment by xam8re** [comment]
This fix multiple file selection on android.

File: ios/Plugin/InAppBrowserPlugin.swift [hunk 1]
```diff
@@ -53,30 +53,33 @@ public class InAppBrowserPlugin: CAPPlugin {
...
+                    return
+                }
+                dataStore.httpCookieStore.getAllCookies { cookies in
+                    let cookiesToDelete = cookies.filter { cookie in
+                        cookie.domain == hostName || cookie.domain.hasSuffix(".\(hostName)") || hostName.hasSuffix(cookie.domain)
+                    }
 
-        guard let url = URL(string: urlString), let hostName = url.host else {
-            call.reject("Invalid URL")
-            return
```
Comment (riderx) [inline ios/Plugin/InAppBrowserPlugin.swift:68]: why changes here? cookie is not related to file choose

File: android/src/main/java/ee/forgr/capacitor_inappbrowser/WebViewDialog.java [hunk 2]
```diff
@@ -111,7 +111,8 @@ public boolean onShowFileChooser(
         ) {
           openFileChooser(
             filePathCallback,
-            fileChooserParams.getAcceptTypes()[0]
```
Comment (riderx) [inline android/src/main/java/ee/forgr/capacitor_inappbrowser/WebViewDialog.java:114]: Do this change still support one file only mode?

Comment (riderx) [reply android/src/main/java/ee/forgr/capacitor_inappbrowser/WebViewDialog.java:114]: ok just saw the comment upper

**Comment by riderx** [comment]
Look good can you just explain why the change in cookie?
if not related please do a separate PR

**Comment by Anthony-Matheou** [comment]
Tried this change out and it's still not working for my purposes in an auth flow. Would it be possible to add my custom solution in the associated [issue](https://github.com/Cap-go/capacitor-inappbrowser/issues/173) as a separate method? Perhaps as a "clearAllCookies" function, since I think mine clears all the cookies for the in app browser? Would be a great help since this is the only Capacitor in-app browser plugin that I've found that has all the other functionality I need for my project. Clearing the cookies/cache is the only functionality I'm missing right now.

**Comment by xam8re** [comment]
> Look good can you just explain why the change in cookie?
> if not related please do a separate PR

I fixed also cookie and forgot to make a specific branch

**Comment by riderx** [comment]
@Anthony-Matheou this PR is supposed to add multi file select, not cookie. 
@xam8re can you remove cookie fix from it?
We do a different PR for this and will test that separately

**Comment by riderx** [comment]
Ok i added both fix myself in separate PR

**Pull request closed without merging**